Frameworks/UIKit/NSTextContainer.mm

```diff
@@ -75,25 +119,69 @@ - (CGRect)lineFragmentRectForProposedRect:(CGRect)proposed
                             remainingRect:(CGRect*)remainingRect {
     CGRect totalRect = CGRectMake(0, 0, _size.width, _size.height);
     CGRect ret = CGRectIntersection(proposed, totalRect);
+    if ([_exclusionPaths count] == 0L || ret.size.width <= 0) {
+        return ret;
+    }
+
+    CGFloat padding = self.lineFragmentPadding;
```

Comment:
> self.lineFragmentPadding; [](start = 21, length = 26)

you have a mix usage of accessing though self and direct access via _xxxxx
standardize.